src/components/form/templates/common.ts

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+export function renderTitle(title: string) {
+    if (!title) {
+        return;
+    }
+
+    return React.createElement(
+        'h1',
+        { className: 'mdc-typography mdc-typography--headline1' },
+        title
+    );
+}
+
+export function renderDescription(description: string) {
+    if (!description) {
+        return;
+    }
+
+    return React.createElement(
+        'p',
+        { className: 'mdc-typography mdc-typography--body1' },
+        description
+    );
+}
+
+/**
+ * Find the first string item in a nested structure.
+ *
+ * If an object has the key 'title' it will have priority, followed by the key 'name'
+ *
+ * @param {*} data the data to find the title for
+ * @param {*} schema schema for the item
+ *
+ * @returns {string} describing title of the data
+ */
+export function findTitle(data: any, schema: any) {
```

Comment:
This should be reasonably easy to write some tests for, right? If so, I think we should.

Don't we already have schema data structures somewhere, that we could use? In lime-admin perhaps? I could swear I've seen similar tests very recently 🙂

Reply:
Well, I guess those were probably tests for the lime-admin schema, so maybe not exactly cut-and-paste to here, but testing this should be very similar I assume?

Reply:
Haha yeah we have more or less the same code in lime-admin, I made som improvements to it here. But we should definitely have tests for it :+1: